app/repositories/Repository.scala

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package repositories
+
+import java.sql.Timestamp
+import java.time.LocalDateTime
+
+import javax.inject.Inject
+import models.paye.PayeAtsMiddleTier
+import org.slf4j.LoggerFactory
+import play.api.libs.json.{JsObject, Json, Reads, __}
+import play.modules.reactivemongo.ReactiveMongoApi
+import reactivemongo.api.indexes.{Index, IndexType}
+import reactivemongo.bson.BSONDocument
+import reactivemongo.play.json.ImplicitBSONHandlers.JsObjectDocumentWriter
+import reactivemongo.play.json.collection.JSONCollection
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import reactivemongo.play.json.collection._
+
+import scala.concurrent.duration.FiniteDuration
+
+class Repository @Inject()(mongo: ReactiveMongoApi) {
+  val logger = LoggerFactory.getLogger("application." + getClass.getCanonicalName)
```

Comment:
loggers never used

Reply:
True dat, Bad habit of mine leaving useful stuff lying around :). I'll get rid